docs/docs/building-ui/element-composite.md

```diff
@@ -163,14 +241,45 @@ ElementEventOptions merged = baseOptions.mergeWith(specificOptions);
 
 ## Interacting with slots {#interacting-with-slots}
```

Comment:
This section needs a bit more explanation about what slots are. You can refer to the MDN documentation for more details.